### PR TITLE
UI refinements in Pending Changes

### PR DESCRIPTION
**Pull request by jgiardino**
- Adds patternfly-react to utilize some of the components in the Pending Changes modal, like Alert and Popover.
- Adds popover with additional info about the changes from a previous session.
- Hides the changes from current session if there are only changes from a previous session.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -24,9 +27,7 @@
     "redux-saga": "^0.15.3",
     "reselect": "^3.0.1",
     "webcomponentsjs": "^1.0.2",
-    "whatwg-fetch": "^1.0.0",
-    "@webcomponents/custom-elements": "^1.0.0",
-    "@webcomponents/webcomponentsjs": "^1.0.0"
+    "whatwg-fetch": "^1.0.0"
```
Comment (henrywang) [inline package.json]: @jgiardino, Are the whatwg-fetch, webcomponents/custom-elements, and webcomponents/webcomponentsjs are required?

**Comment by henrywang** [comment]
@jgiardino, I merged #197. This PR will have code conflict with master. Could you please fix the conflict? Thanks.

File: package.json (hunk 1 above)
Comment (jgiardino) [reply package.json]: My understanding of whatwg-fetch is that it's used for fetching data. I'm assuming we are using it, although I could be wrong.

I added webcomponents/custom-elements and webcomponents/webcomponentsjs when I created the Tabs component. I'd like to replace these Tabs with the patternfly-react Tabs, but will do that in a later PR.

**Comment by henrywang** [comment]
@jgiardino, the new commit failed on eslint test ONLY. Could you please take a look at this issue if you have time. Thanks.
```shell
/home/travis/build/weldr/welder-web/components/Modal/PendingChanges.js
  123:27  error  Expected indentation of 24 space characters but found 26  react/jsx-indent
  133:21  error  Expected indentation of 22 space characters but found 20  react/jsx-indent
```

**Comment by jgiardino** [comment]
Thanks for catching that, @henrywang!

**Pull request merged**